newdle/client/src/components/answer/Calendar.js

```diff
@@ -156,19 +156,24 @@ function Hours({minHour, maxHour, hourStep}) {
   const hourSeries = hourRange(minHour, maxHour, hourStep);
   const hourSpan = maxHour - minHour;
   return (
-    <div className={styles['hours-column']}>
-      {_.range(0, hourSpan + hourStep, hourStep).map((i, n) => (
-        <div
-          className={styles.hour}
-          key={`hour-label-${i}`}
-          style={{top: `${(i / hourSpan) * 100}%`}}
-        >
-          {n === 0 && hourSeries[n] === 0
-            ? '0'
-            : serializeDate(toMoment({hours: hourSeries[n]}), 'k')}
-        </div>
-      ))}
-    </div>
+    <>
+      <Header as="h3" className={styles.date}>
```

Comment:
Is this really necessary? Couldn't we add some padding/margin in `hours-column` instead?